src/test/resources/test.properties

```diff
@@ -1,2 +1,5 @@
 application.version=TEST
-hostInventoryServiceUrl=https://localhost/api/hostinventory
+hostInventoryServiceHostname=test.com
```

Comment:
See previous comments regarding single attribute.

Reply:
REVERTED - Will be included in next commit.